src/js/tech/flash.js

```diff
@@ -268,6 +268,16 @@ class Flash extends Tech {
   }
 
   /**
+   * Get the last error object
+   *
+   * @return {Object}
+   * @method error
+   */
+  error() {
+    return this.error_ || {};
```

Comment:
Heh, oops. Blank object.
If we do, (I don't oppose it), we'd want to fix the call site (https://github.com/videojs/video.js/blob/master/src/js/player.js#L1000-L1002) which assumes an object always.